.gitignore

```diff
@@ -1,2 +1,4 @@
 .env
-node_modules
+**/node_modules/**
```

Comment:
Using `**` as a prefix and postfix to the directory name causes it to be ignored regardless of its location in the file tree.
https://git-scm.com/docs/gitignore explains it more in depth.